### PR TITLE
Setup: Renaming config.plist

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
Although we've got the `config.plist` in **.gitignore**, since we've got a sample file already pushed, all of the local changes are immediately picked up by git, and marked as `diff`.

In this PR we're renaming `config.plist` > `config-demo.plist`, and adding an extra step in the Readme.

@frosty may i bug you with a small PR?
Thanks in advance!

cc @roundhill 

### Testing:
1. Checkout this branch
2. Verify that it won't build right away
3. Rename `Simplenote/config-demo.plist` > `Simplenote/config.plist`
4. Verify that **git status** shows no changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
 * [Xcode](https://developer.apple.com/xcode/).
 
+
+## Setup Credentials
+
+Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid.
+
+In order to create your own Simperium.com Application, please, [head over to our website](https://www.simperium.com) and signup for free.
+
+After you've created your own Simperium application, copy the Sample Config as follows:
```
Comment (frosty) [inline README.md]: I'd write this as:

> After you've created your own Simperium application, copy the sample config. For example:

```
cp Simplenote/config-demo.plist Simplenote/config.plist
```

(let's assume the user is in the top level dir of the git repo)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+
+```
+cp ./Simplenote/Simplenote/config-demo.plist ./Simplenote/config.plist
+```
+
+Then edit the ./Simplenote/config.plist file and change the SPSimperiumAppID and SPSimperiumApiKey fields to the values you got for your app.
```
Comment (frosty) [inline README.md]: > Then edit the new `config.plist` file and change the `SPSimperiumAppID` and `SPSimperiumApiKey` fields to the correct values for your new app.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
+
+This will allow you to compile and run the app on a device or an emulator.
```
Comment (frosty) [inline README.md]: Nitpick: emulator -> simulator

**Comment by jleandroperez** [comment]
Loved ALL of the changes (everything is addressed).

Thanks a lot sir!

**Pull request merged**